addons/web_tour/static/src/js/tip.js

```diff
@@ -316,8 +316,29 @@ var Tip = Widget.extend({
                 of: this.$anchor,
                 collision: "none",
                 using: props => {
-                    this.el.style.setProperty('top', `${props.top}px`, 'important');
-                    this.el.style.setProperty('left', `${props.left}px`, 'important');
+                    const {top} = props;
+                    let {left} = props;
+                    const hasInnerText = this.$anchor.contents().first().text().trim().length;
```

Comment:
This seems too random, you take the first element and check its text. That means that `<div>hello <i class="fa fa-arrow-right"/></div>` will be considered to have inner text but `<div><i class="fa fa-arrow-right"/> hello</div>` will not?

Let's try not to use jquery for this entire code. You might want to rebase this work on the framework refactoring and do not want to use jquery or underscore anymore :wink: 

I am not sure how to make a good condition here, maybe checking the tag of the anchor? "if this is a p, h1, h2, ..., h6" then we consider we have to center on the whole text it contains (not the first text it contains). Otherwise, we keep the old system.